### PR TITLE
Add functionality to also return the actual selected LatLng from Goog…

### LINKED ISSUE #62: Select this location does not return the actual LatLng
Hi Rafael, thank you for this awesome library.

When users click on select this location button, the library does not return the actual LatLng selected by the users. Instead it only return place object retrieved from the Google Place.

It becomes a problem when the app needs the actual coordinate selected by the users. I will submit a PR

### PR DESCRIPTION
**Pull request by mahasadhu**
Pull request proposal for issue #62 

It would be really cool if you could take a look at this this week, since this functionality is quite urgent for me :sweat_smile:

### CODE REVIEW

File: library/src/main/java/com/rtchagas/pingplacepicker/PingPlacePicker.kt [hunk 1]
```diff
@@ -97,6 +105,7 @@ class PingPlacePicker private constructor() {
     companion object {
 
         const val EXTRA_PLACE = "extra_place"
+        const val ACTUAL_SELECTED_LATLNG = "actual_selected_latlng"
```
Comment (rtchagas) [inline library/src/main/java/com/rtchagas/pingplacepicker/PingPlacePicker.kt]: Please follow the naming standards for extras.
Would be nice: EXTRA_ACTUAL_LATLNG = "extra_actual_latlng"

File: library/src/main/java/com/rtchagas/pingplacepicker/ui/PlacePickerActivity.kt [hunk 2]
```diff
@@ -62,6 +62,7 @@ class PlacePickerActivity : AppCompatActivity(), PingKoinComponent,
 
         // For passing extra parameters to this activity.
         const val EXTRA_LOCATION = "extra_location"
+        const val IS_SHOULD_RETURN_ACTUAL_LATLNG = "is_should_return_actual_latlng"
```
Comment (rtchagas) [inline library/src/main/java/com/rtchagas/pingplacepicker/ui/PlacePickerActivity.kt]: Same here: EXTRA_SHOULD_RETURN_ACTUAL_LATLNG

**Review by rtchagas — CHANGES_REQUESTED** [review]
Really nice PR! Thank you!
Just minor comments.

**Comment by mahasadhu** [comment]
@rtchagas variable updated according to naming standards

**Review by rtchagas — APPROVED** [review]
Nice!

**Comment by mahasadhu** [comment]
@rtchagas and thank you for releasing this immediately :smiley: :pray:

**Pull request merged**